Gemfile

```diff
@@ -1,16 +1,26 @@
 source 'https://rubygems.org'
 
-gemspec
-
 group :development do
-  gem 'vagrant',      git: 'https://github.com/mitchellh/vagrant.git'
-  gem 'rake'
+  gem 'vagrant',
+      :git => 'git://github.com/mitchellh/vagrant.git',
+      :ref => 'v1.8.1'
+
+  # test dependencies
+  gem 'minitest'
+  gem 'mocha'
+
   gem 'cucumber', '~> 2.1'
   gem 'aruba', '~> 0.13'
   gem 'komenda', '~> 0.1.6'
   gem 'launchy'
   gem 'gem-compare'
   gem 'mechanize'
+
+  # build tool
+  gem 'rake', '10.4.2'
```

Comment:
And TBH, locking version of Rake due to some warnings is a bit shortsighted because:

1. Its just warnings of test suite, so although they might be annoying, they makes no real harm
2. You will never discover that the warnings were fixed, since you locked the version and using the old version